### PR TITLE
Fix vm errors after save/restore

### PR DESCRIPTION
**Pull request by dzhengfy**
Sometimes vm gets no IP after executing vm save and restore operations.
This may because the vm is not fully started and some services are not
correctly running before executing vm. So this is to fix the issues and
will track if the case is stable in future.

Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

**Comment by dzhengfy** [comment]
@chloerh Could you help review firstly?

**Comment by dzhengfy** [comment]
```
# avocado run --vt-type libvirt --vt-machine-type q35 libvirt.virsh.save.normal_test.id_option.no_option.show_progress,virsh.save.normal_test.normal_option.paused_option.no_progress,virsh.save.normal_test.paused_option.paused_option.show_progress,save.normal_test.acl_test.running_option.show_progress,virsh.save.error_test
JOB ID     : 887bdbe8c8d964691f6ba8cf7edcbcd5a9526892
JOB LOG    : /root/avocado/job-results/job-2020-06-01T11.59-887bdbe/job.log
 (01/14) type_specific.io-github-autotest-libvirt.virsh.save.normal_test.normal_option.paused_option.no_progress: PASS (87.27 s)
 (02/14) type_specific.io-github-autotest-libvirt.virsh.save.normal_test.paused_option.paused_option.show_progress: PASS (60.93 s)
 (03/14) type_specific.io-github-autotest-libvirt.virsh.save.normal_test.acl_test.running_option.show_progress: PASS (84.11 s)
 (04/14) type_specific.io-github-autotest-libvirt.virsh.save.error_test.no_option: PASS (65.95 s)
 (05/14) type_specific.io-github-autotest-libvirt.virsh.save.error_test.hex_id_option: PASS (69.94 s)
 (06/14) type_specific.io-github-autotest-libvirt.virsh.save.error_test.invalid_id_option: PASS (65.33 s)
 (07/14) type_specific.io-github-autotest-libvirt.virsh.save.error_test.unexpect_option: PASS (67.68 s)
 (08/14) type_specific.io-github-autotest-libvirt.virsh.save.error_test.invalid_uuid_option: PASS (66.22 s)
 (09/14) type_specific.io-github-autotest-libvirt.virsh.save.error_test.extra_option: PASS (67.48 s)
 (10/14) type_specific.io-github-autotest-libvirt.virsh.save.error_test.shut_off_option: PASS (33.97 s)
 (11/14) type_specific.io-github-autotest-libvirt.virsh.save.error_test.only_name_option: PASS (65.70 s)
 (12/14) type_specific.io-github-autotest-libvirt.virsh.save.error_test.libvirtd_stop_option: PASS (87.56 s)
 (13/14) type_specific.io-github-autotest-libvirt.virsh.save.error_test.acl_test: PASS (72.60 s)
 (14/14) type_specific.io-github-autotest-libvirt.virsh.save.error_test.readonly: PASS (66.45 s)
RESULTS    : PASS 13 | ERROR 1 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 1311.22 s

```

File: libvirt/tests/src/virsh_cmd/domain/virsh_save.py [hunk 1]
```diff
@@ -78,6 +87,7 @@ def run(test, params, env):
         utils_libvirtd.libvirtd_start()
 
     if savefile:
+        time.sleep(10)
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/domain/virsh_save.py:90]: Is there any way to not hardcode wait time,and use uitls_mic.waitfor(xx) to implement this?

**Comment by qiankehan** [comment]
From the description, it seems that the reason of vm errors are not clear and the fix is a possible workaround.
Before the fix, these questions should be answered:
1. What's the reason of these error? Can they be reproduced by manual steps or shell scripts?
2. If the reason is what you guessed in the commit msg, is there any other way to tell if vm is fully started instead of just waiting?

File: libvirt/tests/src/virsh_cmd/domain/virsh_save.py (hunk 1 above)
Comment (kylazhang) [reply libvirt/tests/src/virsh_cmd/domain/virsh_save.py:90]: Why sleep before restore?

**Comment by dzhengfy** [comment]
I'd like to hold on this PR's review now because I am wondering if it is a must for vm to start completely before save. I guess not now, but I need to get a confirmation.

**Pull request closed without merging**